modules/lang-groovy/src/test/java/org/elasticsearch/messy/tests/SimpleSortTests.java

```diff
@@ -1650,8 +1650,8 @@ public void testNestedSort() throws IOException, InterruptedException, Execution
             assertThat(hits[i].getSortValues().length, is(1));
             Object o = hits[i].getSortValues()[0];
             assertThat(o, notNullValue());
-            assertThat(o instanceof StringAndBytesText, is(true));
-            StringAndBytesText text = (StringAndBytesText) o;
+            assertThat(o instanceof Text, is(true));
```

Comment:
I'm just used to `ClassCastException`s  in tests being test failures. The cast is implicitly part of the test. I'm fine with keeping the explicit assertion but I'd prefer the other style so you get nicer error messages.